tests/testConnectionPassing.py

```diff
@@ -0,0 +1,194 @@
+                                  % (ix, cls.star_ra[ix], cls.star_dec[ix],
+                                     cls.star_umag[ix], cls.star_gmag[ix]))
+
+    @classmethod
+    def write_galaxy_txt(cls):
+        np.random.seed(88)
+        cls.n_galaxies = 100
+        cls.gal_ra = np.random.random_sample(cls.n_galaxies)*360.0
+        cls.gal_dec = (np.random.random_sample(cls.n_galaxies)-0.5)*180.0
+        cls.gal_redshift = np.random.random_sample(cls.n_galaxies)*5.0
+        cls.gal_umag = np.random.random_sample(cls.n_galaxies)*10.0+21.0
+        cls.gal_gmag = np.random.random_sample(cls.n_galaxies)*10.0+21.0
+
+        cls.gal_txt_name = os.path.join(getPackageDir('sims_catalogs_measures'),
+                                         'tests', 'scratchSpace',
+                                         'ConnectionPassingTestGal.txt')
+
+        if os.path.exists(cls.gal_txt_name):
+            os.unlink(cls.gal_txt_name)
+
+        with open(cls.gal_txt_name, 'w') as output_file:
+            output_file.write("#id raJ2000 decJ2000 redshift umag gmag\n")
+            for ix in range(cls.n_galaxies):
+                output_file.write("%d %.4f %.4f %.4f %.4f %.4f\n"
+                                  % (ix, cls.gal_ra[ix], cls.gal_dec[ix],
+                                     cls.gal_redshift[ix],
+                                     cls.gal_umag[ix], cls.gal_gmag[ix]))
+
+
+    @classmethod
+    def setUpClass(cls):
+
+        cls.write_star_txt()
+        cls.write_galaxy_txt()
+
+        cls.dbName = os.path.join(getPackageDir('sims_catalogs_measures'), 'tests',
+                                  'scratchSpace', 'ConnectionPassingTestDB.db')
+
+        if os.path.exists(cls.dbName):
+            os.unlink(cls.dbName)
+
+        galDtype = np.dtype([('id', np.int),
+                             ('raJ2000', np.float), ('decJ2000', np.float),
+                             ('redshift', np.float), ('umag', np.float),
+                             ('gmag', np.float)])
+
+        starDtype = np.dtype([('id', np.int), ('raJ2000', np.float),
+                              ('decJ2000', np.float), ('umag', np.float),
+                              ('gmag', np.float)])
+
+
+        dbo = fileDBObject(cls.star_txt_name,
+                           database=cls.dbName, driver='sqlite',
+                           runtable='stars', idColKey='id',
+                           dtype=starDtype)
+
+        dbo = fileDBObject(cls.gal_txt_name,
+                           database=cls.dbName, driver='sqlite',
+                           runtable='galaxies', idColKey='id',
+                           dtype=galDtype)
+
+
+
+    @classmethod
+    def tearDownClass(cls):
+        if os.path.exists(cls.dbName):
+            os.unlink(cls.dbName)
+
+        if os.path.exists(cls.star_txt_name):
+            os.unlink(cls.star_txt_name)
+
+        if os.path.exists(cls.gal_txt_name):
+            os.unlink(cls.gal_txt_name)
+
+
+    def test_passing(self):
```

Comment:
Ok, I think this answers my question from the sims_catalogs_generation pull request.